source/textmate_tools.rb

```diff
@@ -136,16 +137,67 @@ def recursively_set_each_value!(a_lambda)
 
 
 
-
 class Grammar
-    attr_accessor :data, :all_tags, :language_ending
+    attr_accessor :data, :all_tags, :language_ending, :namespace, :export_options
     
-    #
-    # Globally accessible current grammar object
-    #
-    @@current_grammar = nil
-    def self.current_grammar
-        return @@current_grammar
+    # 
+    # import and export methods
+    # 
+    @@export_data
+    def self.export(*args, &block)
```

Comment:
So an export always has to be followed by an import, right?
export, export, import, import won't work?

Reply:
Yeah that would fail. The idea is to use Grammar.export somewhat similarly to how node.js uses module.export. 

So basically, if you want to export a textmate pattern, wrap the entire file inside of a 
```
Grammar.export() do 
end
```
And then return the things you want to export.

Reply:
**However, every time `import` gets called, it should run the file, which will assign the Grammar.export. 
So `export, export, import, import` would be more like `export, export, import(export), import(export)`